### PR TITLE
"Islandora 8" audit

### PR DESCRIPTION
**Pull request by rosiel**
[Edit to add ticket:] https://github.com/Islandora/documentation/issues/1984

## Purpose / why

Islandora 8 appears a lot in the documentation

## What changes were made?

going through some pages to remove "Islandora 8" and replace it with "Islandora" vs Legacy.

## Verification

Does this still make sense?

## Interested Parties

> _**Replace this text** - Name some folks who may be interested, if documentation related mention @Islandora/documentation, or, if unsure, @Islandora/committers_

---

## Checklist

> __Pull-request reviewer__ should ensure the following

* [x] Does this PR link to related [issues](https://github.com/Islandora/documentation/issues/)?
* [ ] Does the proposed documentation align with the [Islandora Documentation Style Guide](https://islandora.github.io/documentation/contributing/docs_style_guide/)?
* [ ] Are the changes accurate, useful, free of typos, etc?

> __Person merging__ should ensure the following
* [ ] Does mkdocs still build successfully? (This is indicated by TravisCI passing. To test locally, and see warnings, see [How To Build Documentation](https://islandora.github.io/documentation/technical-documentation/docs-build/).)
* [x] If pages are renamed or removed, have all internal links to those pages been fixed?
* [x] If pages are added, have they been linked to or placed in the menu?
* [ ] Did the PR receive at least one approval from a committer, and all issues raised have been addressed?

### CODE REVIEW

**Review by seth-shaw-unlv — CHANGES_REQUESTED** [review]
Just one little nit-pick, we should be consistent using "Markdown" (this is the form used by [the original creator](https://daringfireball.net/projects/markdown/)) instead of "MarkDown".

Otherwise, this all looks good to me.

**Comment by rosiel** [comment]
> Just one little nit-pick, we should be consistent using "Markdown" (this is the form used by [the original creator](https://daringfireball.net/projects/markdown/)) instead of "MarkDown".
> 
> Otherwise, this all looks good to me.

Don't get me started on OpenSeadragon...

**Review by seth-shaw-unlv — APPROVED** [review]

**Comment by seth-shaw-unlv** [comment]
So far so good.

**Comment by rosiel** [comment]
There are a lot of files to go but I'm going to trim this here and do further PRs for the next ones. (advice would be welcomed, too. It's so tempting to proofread while I have the files open but I might have to stop in the name of getting it done)

File: docs/contributing/editing-docs.md [hunk 1]
```diff
@@ -10,15 +10,17 @@ The [Islandora Documentation Interest Group](https://github.com/islandora-intere
...
-- have a basic familiarity with [Markdown](http://en.wikipedia.org/wiki/Markdown), or a good [MarkDown cheat sheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet).
-    - for example, you may use MarkDown syntax to create links and section headings.
+    - Github accounts are free and only require a valid email address.
+- have a very basic familiarity with Git terms such as [fork](https://guides.github.com/activities/forking/), [branch](https://guides.github.com/activities/hello-world/#branch), and [pull request](https://guides.github.com/activities/hello-world/#pr).
+- optionally, be a member of the Islandora [GitHub Organization](https://github.com/orgs/Islandora/people).
+    - To request an invite, email community@islandora.ca or ask on the Islandora Slack.
+    - This is only needed if you want to commit changes directly to the documentation. Non-members may still make pull requests. 
+- be willing to learn with [Markdown](http://en.wikipedia.org/wiki/Markdown) - a good [Markdown cheat sheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) can help.
```
Comment (ysuarez) [inline docs/contributing/editing-docs.md]: Did you mean to leave the word "with" in line 18? I can make my own PR to fix that and run a CLI spellchecker while I am at it.

**Comment by ysuarez** [comment]
Should we add entries in the glossary for "Islandora Legacy" and/or "Islandora 8" too to help others with thsi transition in terms?  I can create a new PR for it or try to push to @rosiel PR?

**Comment by misilot** [comment]
@rosiel I opened a [PR](https://github.com/rosiel/documentation/pull/1) against your branch if you want to take a look at that.

**Comment by rosiel** [comment]
Thanks Tom. The reason I was doing it manually was for catching things like "Islandora 7 and 8" and leaving "Islandora 8" if it's in the title of a recorded youtube video or presentation. I was also doing a mild sanity check as I went, since this update is going to change the "last edited" date. That's why i stopped at 12 files. Let me bring this back to "Draft" status and pull in your changes.

**Comment by misilot** [comment]
Oh, sorry!

**Comment by rosiel** [comment]
Thank you so much @misilot ! Your help was a huge time-saver.  @ysuarez I love the suggestion, and just added it.

**Comment by misilot** [comment]
Glad it was was helpful in the end! Sorry you had to change "OpenSeadragon" back to "Open Seadragon".  I had gone with the spelling/formatting from https://openseadragon.github.io/

**Comment by rosiel** [comment]
That's correct. But as docs, I think they should be descriptive of what the user sees in the UI. That's why I was careful to leave the module name and file formatter as OpenSeadragon while the Context, Media Display Mode, Node Display Mode, and Taxonomy Term, are all as they appear, wrongly, Open Seadragon.

File: docs/contributing/committers.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+Islandora is open source and released under MIT and GPLv2 licenses. The software and associated documentation is developed collectively by a community of contributors and committers. All interested community members are encouraged to contribute to the project. Contributors who demonstrate sustained engagement with the project through quality participation in meetings, mailing lists, documentation and code updates can be nominated by existing committers to also become a committers. It should be emphasized that committers need not be limited to software developers. Community members with skills in documentation and testing, for example, can also be committers. We maintain a separate list of [Islandora Legacy Committers](https://github.com/Islandora/islandora/wiki/Islandora-Committers)
```
Comment (seth-shaw-unlv) [inline docs/contributing/committers.md:3]: Nit-pick: missing a period at the end of the paragraph.

File: docs/installation/docker-troubleshooting.md [hunk 3]
```diff
@@ -26,4 +26,4 @@ ERROR: for isle-dc_mariadb_1  UnixHTTPConnectionPool(host='localhost', port=None
 
 you can try quitting Docker completely (make sure there is no whale icon in your top toolbar - may need to select "Quit" from the whale icon itself) and then restart Docker.
 ### 504 Bad Gateway
-If you get a white screen with a 504 Bad Gateway error, this means your containers haven't finished initializing themselves yet.  If you've waiting an appropriate amount of time (2-5 minutes), then there is most likely an error in a container's startup script.  Use `docker ps -a` to see which services have `Exited` status, and then tail their logs with `docker-compose logs service_name`.
+If you get a white screen with a 504 Bad Gateway error, this means your containers haven't finished initializing themselves yet.  If you've waiting an appropriate amount of time (2-5 minutes), then there is most likely an error in a container's startup script.  Use ` docker ps -a` to see which services have `Exited` status, and then tail their logs with `docker-compose logs service_name`.
```
Comment (seth-shaw-unlv) [inline docs/installation/docker-troubleshooting.md:29]: There shouldn't be a space between "`" and "docker".

**Comment by seth-shaw-unlv** [comment]
@rosiel , just two small adjustments then 👍.

**Comment by misilot** [comment]
@seth-shaw-unlv I made the changes if you want to review at https://github.com/rosiel/documentation/pull/2

**Comment by seth-shaw-unlv** [comment]
That is one way to do it.

**Pull request merged**